index.js

```diff
@@ -100,11 +101,20 @@ function addExceptionsToUnprotected (exceptions) {
 }
 
 const listData = JSON.parse(fs.readFileSync(`${LISTS_DIR}/${unprotectedListName}`))
-defaultConfig.unprotectedTemporary = listData.exceptions
-
-addExceptionsToUnprotected(defaultConfig.unprotectedTemporary)
+addExceptionsToUnprotected(listData.exceptions)
 addExceptionsToUnprotected(defaultConfig.features.contentBlocking.exceptions)
 
+const notExcluded = ['adClickAttribution', 'appTrackerProtection', 'autofill', 'duckPlayer', 'incontextSignup',
```

Comment:
`notExcluded` could be read either way as not excluded from this treatment or not an exclusion.
Could we consider using a name that adds a little more clarity like `excludedFeaturesFromUnprotectedTemporaryExceptions` (maybe there's something less wordy it's late here ha)

Reply:
Didn't have any kind of brainstorm on shortening that aside from `Temp` for `Temporary` assuming we want to keep all the context 🤷‍♀️